src/main/java/org/openmrs/maven/plugins/AbstractTask.java

```diff
@@ -62,6 +64,20 @@
     Wizard wizard;
 
     /**
+     * Interactive mode param
+     *
+     * @parameter expression="${interactiveMode}" default-value=true
+     */
+    String interactiveMode;
```

Comment:
It's not obvious what it does. Is it for testing only? If yes, then please add a comment.